tests/scripts/actionutils.sh

```diff
@@ -256,17 +256,18 @@ function remote_enable_rbd_mirror_daemon() {
 function remote_wait_for_secondary_to_sync() {
     set -eux
 
-    # wait till images are synchronised
+    # wait till $1 images are synchronised
+    local threshold="${1?missing}"
     count=0
     for index in {1..100}; do
         echo "Check run #$index"
-        list_output=$(lxc exec node-wrk2 -- sh -c "sudo microceph remote replication rbd list --json")
+        list_output=$(lxc exec node-wrk2 -- sh -c "sudo microceph replication list rbd --json")
         echo $list_output
-        images=$(echo $list_output | jq .[].Images)
+        images=$(echo $list_output | jq length .[].Images)
```

Comment:
Don't think jq length works here. Possibly just count output lines with `| wc -l`?

Reply:
yeah, i patched it out with some added awk. waiting for CI.